### PR TITLE
Fix: Upgrade yourkit to b139 to fix broken develop

### PR DESCRIPTION
**Pull request by robert3005**
## Before this PR
You can't run build on develop

## After this PR
==COMMIT_MSG==
Upgrade yourkit agent version to avoid broken develop
==COMMIT_MSG==

## Possible downsides?
N/A

### CODE REVIEW

File: gradle-sls-packaging/build.gradle [hunk 1]
```diff
@@ -45,7 +45,7 @@ pluginBundle {
     }
 }
 
-def yourkitVersion = "2019.8-b137"
+def yourkitVersion = "2019.8-b139"
```
Comment (ferozco) [inline gradle-sls-packaging/build.gradle:48]: you have to update the sha :(

**Comment by ferozco** [comment]
We've been periodically making these manual PRs for a while now. It would be really great if we could automate this

File: gradle-sls-packaging/build.gradle (hunk 1 above)
Comment (robert3005) [reply gradle-sls-packaging/build.gradle:48]: just did

**Comment by robert3005** [comment]
tbh I think we should rely on jfr instead of yourkit agent. There's a lot more infrastructure around that than yourkit

**Comment by ferozco** [comment]
👍

**Pull request merged**